client/src/templates/Challenges/utils/frame.js

```diff
@@ -61,9 +61,12 @@ export const runTestInTestFrame = async function (document, test, timeout) {
   ]);
 };
 
-const createFrame = (document, id) => ctx => {
+const createFrame = (document, id, title) => ctx => {
   const frame = document.createElement('iframe');
   frame.id = id;
+  if (typeof title === 'string') {
+    frame.title = title;
+  }
```

Comment:
Quick question, is this check to ensure we don't assign null to `frame.title`, or the value of `title` can be in other types?

Reply:
There is one instance in which this function will be called and the `title` parameter will not be a string (the `createTestFramer` function does not pass in a `title` parameter since it is not seen by the user, and thus the `title` parameter will be `undefined`). I could have checked for that, but the `title` attribute on the frame should be a string, so I decided to ensure it was a string instead.

Reply:
Gotcha. We could also use the truthiness check (`if (title) { // do something }`) but what you have here is safer, especially when this file has not been converted to TypeScript yet.